packages/block-library/src/missing/edit.native.js

```diff
@@ -252,5 +254,4 @@ export default compose( [
 		};
 	} ),
 	withPreferredColorScheme,
-	withUIStrings,
```

Comment:
Thanks, that's the part I prefer here, I think `withUIStrings` would have been a bit confusing for the web.